tasks/yum.yml

```diff
@@ -11,3 +11,12 @@
   - maven
   - patch
   - MySQL-python
+  - python-devel
+  - libffi-devel
+- name: packages used by the python part of the repox configuration/installation
+  pip:
```

Comment:
This `yum.yml` file is now misnamed.  I should have called it `packages.yml` or something instead.